src/reader-activation-auth/accessibility.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Trap focus in the modal when opened.
+ * See: https://uxdesign.cc/how-to-trap-focus-inside-modal-to-make-it-ada-compliant-6a50f9a70700
+ */
+export function trapFocus( currentModal ) {
+	const focusableEls =
+		'button, [href], input:not([type="hidden"]), select, textarea, [tabindex]:not([tabindex="-1"])';
+	const visibleFocusableEls = [];
+
+	const firstFocusableEl = currentModal.querySelectorAll( focusableEls )[ 0 ]; // get first element to be focused inside modal
+	const focusableElsAll = currentModal.querySelectorAll( focusableEls );
```

Comment:
Non-blocking nit: Is there a reason we are running the same call to `querySelectorAll` here? Seems like we can run it once, then just get the first element.

Reply:
Oh yeah, that's not great -- this has been updated!